packages/components/src/icon-button/style.scss

```diff
@@ -9,13 +9,14 @@
 	position: relative;
 	overflow: hidden;
 	text-indent: 4px;
+	border-radius: $radius-round-rectangle;
```

Comment:
Does this change make the following line redundant now?

https://github.com/WordPress/gutenberg/blob/c1cdd95753ba1b0f36065bd9bd9fc5bc194021af/edit-post/components/header/style.scss#L55

Reply:
Also, I don't understand why it's IconButton which applies hover and active button styles (and the associated border radius), rather than the underlying Button ?

See related comment at https://github.com/WordPress/gutenberg/issues/7534#issuecomment-419543749

Reply:
Yes, that makes this rule redundant.

And I don't have a good answer to your question, other than this is how we got to where we are today. 

I very much appreciate the effort you are putting in https://github.com/WordPress/gutenberg/pull/9702 — would you like help with that? Do you want to remove the redundant rule in the header as part of that PR, or would you like me to create a quick fixup here?

Reply:
There doesn't appear to be great consensus around #9702 / #7534, so if the change can happen separately, it's probably better.

Reply:
Created PR here: https://github.com/WordPress/gutenberg/pull/10021